### ISSUE TITLE
Silent failure on `draft up` with bad registry.token

### ISSUE THREAD

**Issue by frodopwns** [Azure/draft#162]
I starting playing with draft today. During the learning process I ran into one event that may be a bug.

Ubuntu 16.10
Helm 2.4.1
Draft Canary (18784fd0052047288a4bd5e32df9929d70469f35)

After running `draft init` and then `draft init --set --upgrade` to set the config items (I later learned I had set the wrong token for docker.io) I got this output:

```
~/go/src/github.com/Azure/draft/examples/python on  master! :watch: 17:49:18
$ draft up
--> Building Dockerfile
Step 1 : FROM python:onbuild
onbuild: Pulling from library/python
ef0380f84d05: Pulling fs layer
24c170465c65: Pulling fs layer
4f38f9d5c3c0: Pulling fs layer
4125326b53d8: Pulling fs layer
35de80d77198: Pulling fs layer
ea2eeab506f8: Pulling fs layer
1c7da8f3172e: Pulling fs layer
e30a226be67a: Pulling fs layer
d91897da0dfd: Pulling fs layer
4125326b53d8: Waiting
35de80d77198: Waiting
ea2eeab506f8: Waiting
1c7da8f3172e: Waiting
e30a226be67a: Waiting
d91897da0dfd: Waiting
24c170465c65: Verifying Checksum
24c170465c65: Download complete
ef0380f84d05: Verifying Checksum
ef0380f84d05: Download complete
35de80d77198: Verifying Checksum
Watching local files for changes...
```

The part that I expected but was missing is:
```
Error encountered streaming JSON response: unauthorized: authentication required
Watching local files for changes...
```

I was unable to recreate this problem.

**Comment by bacongobbler** [comment]
I have a feeling this will be fixed with the rewrite to grpc in #1. pinging @fibonacci1729 for visibility

**Comment by bacongobbler** [comment]
closing this as semi-fixed via #154, then also fixed via the PR incoming to rewrite this via grpc.

**Comment by squillace** [comment]
@bacongobbler you saw that I had this problem also. SWAG for grpc rewrite? Just curious.
